src/translib/transformer/xfmr_vrf.go

```diff
@@ -45,6 +45,7 @@ var NwInstTblNameMapWithName = map[string]string {
 	"mgmt": "MGMT_VRF_CONFIG",
 	"Vrf": "VRF",
 	"default": "VRF",
+    "Vlan": "VLAN",
```

Comment:
for the create case from top level, may also need to update NwInstTblNameMapWithNameAndType